machine-learning/tutorials/GitHubIssueClassification/GitHubIssueClassification.csproj

```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="0.10.0" />
+    <PackageReference Include="Microsoft.ML" Version="0.11.0-preview-27428-4" />
```

Comment:
Is preview version needed?

Reply:
Hi @NextTurn! We are in process of updating this sample for version 0.11.0, and are using the preview version to make initial changes. After 0.11.0 release, we'll update this with the released version, make another review pass, update the tutorial, and push to live.